google-cloud-bigquery/acceptance/bigquery/advanced_test.rb

```diff
@@ -109,6 +109,66 @@
     assert_equal -Float::INFINITY, row[:negative_infinity]
   end
 
+  it "executes SQL with multiple statements and creates child jobs with script_statistics" do
+    multi_statement_sql = <<~SQL
+    -- Declare a variable to hold names as an array.
+    DECLARE top_names ARRAY<STRING>;
+    -- Build an array of the top 100 names from the year 2017.
+    SET top_names = (
+    SELECT ARRAY_AGG(name ORDER BY number DESC LIMIT 100)
+    FROM `bigquery-public-data.usa_names.usa_1910_current`
+    WHERE year = 2017
+    );
+    -- Which names appear as words in Shakespeare's plays?
+    SELECT
+    name AS shakespeare_name
+    FROM UNNEST(top_names) AS name
+    WHERE name IN (
+    SELECT word
+    FROM `bigquery-public-data.samples.shakespeare`
+    );
+    SQL
```

Comment:
Nit: This block would be more readable if the content (between `<<~SQL` and the closing `SQL`) were indented two more spaces.

Same for the examples in comments below.

Reply:
good idea, i'll do it.

Reply:
Done.